go/vt/vtctl/grpcvtctldserver/server.go

```diff
@@ -673,6 +673,37 @@ func (s *VtctldServer) CleanupSchemaMigration(ctx context.Context, req *vtctldat
 	return resp, nil
 }
 
+// ForceCutOverSchemaMigration is part of the vtctlservicepb.VtctldServer interface.
+func (s *VtctldServer) ForceCutOverSchemaMigration(ctx context.Context, req *vtctldatapb.ForceCutOverSchemaMigrationRequest) (resp *vtctldatapb.ForceCutOverSchemaMigrationResponse, err error) {
+	span, ctx := trace.NewSpan(ctx, "VtctldServer.ForceCutOverSchemaMigrationResponse")
```

Comment:
This should just be the function name, not the response part.

Reply:
I don't follow, can you please explain?

Reply:
The trace span name should be the function name:
```
span, ctx := trace.NewSpan(ctx, "VtctldServer.ForceCutOverSchemaMigration")
```